best-practices/pair_programming.md

```diff
@@ -0,0 +1,21 @@
+# Pair Programming
+
+Pair programming is an technique that allows developers to closely collaborate on
```

Comment:
Typo: `an technique`.

Reply:
I'm pretty sure that's correct in en-GB....will ask Julian 😆